docs/source/en/conceptual/stable_diffusion.mdx

```diff
@@ -1,15 +1,332 @@
+
+``` python                                                                                                                                                                                                                                    
+pipe.enable_attention_slicing()                                                                                                                                                                                                               
+```                                                                                                                                                                                                                                           
+
+Great now that attention slicing is enabled, let's try to double the batch size again, going for `batch_size=8`.                                                                                                                              
+
+``` python                                                                                                                                                                                                                                    
+images = pipe(**get_inputs(batch_size=8)).images                                                                                                                                                                                              
+image_grid(images, rows=2, cols=4)                                                                                                                                                                                                            
+```                                                                                                                                                                                                                                           
+
+![img](https://huggingface.co/datasets/diffusers/docs-images/resolve/main/stable_diffusion_101/sd_101_5.png)                                                                                                                                  
+
+Nice, it works. However, the speed gain is again not very big (it might however be much more significant on other GPUs).                                                                                                                      
+
+We're at roughly 3.5 seconds per image 🔥 which is probably the fastest we can be with a simple T4 without sacrificing quality.                                                                                                               
+
+Next, let's look into how to improve the quality!                                                                                                                                                                                             
```

Comment:
Maybe mention `xformers`, even if we don't use it in the notebook?

Reply:
I mentioned it in the very end :-)